content/blog/using-the-power-of-mdx-with-tina.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Using the power of MDX with Tina
+date: '2021-12-09T12:09:32-04:00'
+author: James Perkins
+last_edited: '2021-12-09T12:09:32-04:00'
+---
+# Using the power of MDX with Tina
+
+Tina allows content teams and developers to work at a fast pace and removes the friction between static sites and editing content. We took this approach a step further with the release of our MDX support, which allows developers to create reusable components and content teams to use them whenever they need to. This blog post will show you how to add Tina to your site, then how to create and use components to Tina.
+
+### Project setup.
+
+Step 1: Create a project with tailwind
+``` bash,copy
+npx create-next-app -e with-tailwindcss tina-demo
+cd tina-demo
+```
+Step 2: Add Tina to the project
+
+Use the following command inside of the project to add all the Tina dependencies and wrap the application, ready to be used.
+
+``` bash,copy
+npx @tinacms/cli@latest init
```

Comment:
Should they choose to override the _app.js when this command asks them?

Reply:
Yup. I should add that.